src/Bundle/Form/DataTransformer/ResourceToIdentifierCacheableTransformer.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\Form\DataTransformer;
```

Comment:
```suggestion
namespace Sylius\Resource\Symfony\Form\DataTransformer;
```
So it should be placed into src/Component/src/Symfony/Form/DataTransformer directory.